scripts/superdesk/editor2/add-content.js

```diff
@@ -132,7 +132,7 @@ function AddContentCtrl (scope, element, superdesk, editor, $timeout, config, $q
             addPicture: function() {
                 superdesk.intent('upload', 'media').then(function(images) {
                     $q.all(images.map(function(image) {
```

Comment:
I don't think so, because we are in `addPicture`. We are adding only picture there